### ISSUE TITLE
AttributeError: module 'weeutil' has no attribute 'Moon'

### ISSUE THREAD

**Issue by hoetzgit** [chaunceygardiner/weewx-celestial#1]
Hello,

I had to add the following line:

`import weeutil.Moon`

The reason was the following error message:

```
Traceback (most recent call last):
  File "/home/weewx/bin/wee_reports", line 149, in <module>
    main()
  File "/home/weewx/bin/wee_reports", line 113, in main
    engine = weewx.engine.DummyEngine(config_dict)
  File "/home/weewx/bin/weewx/engine.py", line 93, in __init__
    self.loadServices(config_dict)
  File "/home/weewx/bin/weewx/engine.py", line 161, in loadServices
    obj = weeutil.weeutil.get_object(svc)(self, config_dict)
  File "/home/weewx/bin/user/celestial.py", line 97, in __init__
    self.moon_phases = weeutil.Moon.moon_phases
AttributeError: module 'weeutil' has no attribute 'Moon'

```

Thank you very much for your great WeeWX extensions and best regards
Henry

**Comment by chaunceygardiner** [comment]
I cannot reproduce this error and the line numbers don’t match up with the latest code:
“ self.moon_phases = weeutil.Moon.moon_phases” is on line 96, but your error message has it on line 97.

Would you try again with code exactly as is in the repository and post any errors you get?

**Comment by hoetzgit** [comment]
Hello,

I have made only one modification, otherwise the extension cannot be executed on my system:

Line `32` 
from
`if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 9):`
to
`if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] < 8):
`

My Pyhton version is `3.8.10`, WeeWX Version is `4.9.2a1`

Here is my error message from this morning

```
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****  module 'weeutil' has no attribute 'Moon'
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****  Traceback (most recent call last):
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****    File "/home/weewx/bin/weewxd", line 148, in main
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****      engine = weewx.engine.StdEngine(config_dict)
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****    File "/home/weewx/bin/weewx/engine.py", line 93, in __init__
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****      self.loadServices(config_dict)
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****    File "/home/weewx/bin/weewx/engine.py", line 161, in loadServices
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****      obj = weeutil.weeutil.get_object(svc)(self, config_dict)
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****    File "/home/weewx/bin/user/celestial.py", line 96, in __init__
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****      self.moon_phases = weeutil.Moon.moon_phases
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****  AttributeError: module 'weeutil' has no attribute 'Moon'
Jan 13 07:52:36 weewx weewx[915639] CRITICAL __main__:     ****  Exiting
```

If I add this line, the extension works without any errors visible to me

`import weeutil.Moon`

Henry

**Edit**: Here is the modified code [GitHub](https://github.com/hoetzgit/weewx-pwsweiherhammer/blob/master/home/weewx/bin/user/celestial.py)

**Comment by chaunceygardiner** [comment]
I believe I use features of date that are in Python 3.9; but if it works for you, that’s great.  Closing because this issue is not reproducible with the release.  (BTW, it also make no sense in that in Python, if one imports weeutil, then a reference to weeutil.Moon is valid.)

**Comment by hoetzgit** [comment]
ok, thank you. This was my sample for the include 

Line 18:
https://github.com/weewx/weewx/blob/master/bin/weewx/almanac.py

**Comment by chaunceygardiner** [comment]
I can’t explain why I don’t have the exception and you do (with only the Python version differing).  I’ll change the import weeutil to import weeutil.Moon and re-release.  Re-opening until I test and release the change.

**Comment by chaunceygardiner** [comment]
Changed import weeutil to import weeutil.Moon and released as v0.7.
